src/main/java/org/apache/maven/plugin/patch/ApplyMojo.java

```diff
@@ -278,7 +289,14 @@ public void execute()
                         + patchDirectory );
                 }
 
-                List foundPatchFiles = FileUtils.getFileNames( patchDirectory, "*", null, false );
+                String excludePatterns = "";
```

Comment:
```suggestion
                String excludePatterns = null;
```
This way it works like the original code.